src/js/mediaelement-i18n-locale-pl.js

```diff
@@ -33,10 +33,10 @@
 			"mejs.fullscreen": "Pełny ekran",
 
 			// mediaelementplayer-feature-jumpforward
-			//"mejs.time-jump-forward": "Jump forward %1 second(s)",
+			"mejs.time-jump-forward": "Przewiń do przodu o %1 sek.",
```

Comment:
According to the rules of plurals, it should be 3 translations, and I only see one, can you add the other 2 please?